sklearn/ensemble/tests/test_stacking.py

```diff
@@ -52,6 +55,10 @@
 X_diabetes, y_diabetes = diabetes.data, diabetes.target
 iris = load_iris()
 X_iris, y_iris = iris.data, iris.target
+X_multilabel_r, y_multilabel_r = make_multilabel_classification(
```

Comment:
By the way, do you know the motive for the `_r` suffix?

Reply:
I must admit I don't 😅

Reply:
I guess, we can strip it.